packages/components/src/composite/stories/index.story.tsx

```diff
@@ -353,3 +354,50 @@ const Fill = ( { children } ) => {
 		},
 	},
 };
+
+/**
+ * Combining the `Tooltip` and `Composite` component has a few caveats.
+ * For example, the following code won't work as expected:
```

Comment:
I wonder if it is a good idea to show a snipped that doesn't work as expected 🤔

Reply:
FWIW, it's not immediately clear what doesn't work and why. Should we further elaborate on that?

Reply:
I think it makes sense to show a broken snippet if it's a common mistake. A good way to distinguish it is by adding comments to the snippet, along with some emojis:

```jsx
// 🔴 Does not work
<Composite.Item
...

// 🟢 Good
<Composite.Item
...
```

This should help those who scan code snippets without reading the descriptions.

Example: https://react.dev/learn/you-might-not-need-an-effect